frontend/src/components/contact/ContactManagerView/ContactResultComponent/__snapshots__/ContactResultComponent.test.tsx.snap

```diff
@@ -984,7 +984,11 @@ exports[`Contact Search Results Table matches snapshot 1`] = `
   >
     <div
       class="align-self-center col-auto"
-    />
+    >
+      <span>
+        1 - 3 of  3
+      </span>
```

Comment:
Is this because is defaulting to showing the range now?

Reply:
not exactly defaulting to show the range, before if the showSelectedRowCount boolean would be ignored unless the pageSize and total results was passed in. Since that is now calculated if not specified, the showSelectedRowCount boolean actually controls the display of that element.